### PR TITLE
[alarmdecoder] allow keypad messages with !KPM:

### PR DESCRIPTION
**Pull request by ddempsey3**
[alarmdecoder] allow keypad messages with !KPM:

This PR updates the alarmdecoder binding to accept KeyPad messages with the `!KPM` prefix.  

Logs from my serial AD2USB, which indicate the prefix.
```
10:41:44.219 [TRACE] [oder.internal.handler.ADBridgeHandler] - Received msg: !KPM:[10000001100000003A--],008,[f70000051008001c28020000000000]," DISARMED CHIME   Ready to Arm  "
10:41:48.178 [TRACE] [oder.internal.handler.ADBridgeHandler] - Received msg: !KPM:[10000001100100000A--],011,[f70000051011005028020000000000],"LOBAT 11                        "

```

I was able to verify the changes here enable the KeyPad.

Thread started.
https://community.openhab.org/t/alarmdecoder-2-5-7-binding-serial-keypad/103242

### CODE REVIEW

File: bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java
Comment (bobadair) [inline bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java:59]: I can't say for sure that the keypad message string will never include a ":" character, so doing a split on ":" could potentially cause a problem at some point. It may be better to use something like ```message.replace("!KPM:", "")```.

**Comment by bobadair** [comment]
Thanks @ddempsey3! This is a great addition. I really just have one comment above. Other than that, it looks good to me.

File: bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java
Comment (ddempsey3) [reply bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java:59]: Sure thing, will change!!

**Comment by berndpfrommer** [comment]
I was requested as a reviewer for the PR, but I have not looked at the code in a long while, would have to read up on it again from scratch. I have sold my house including alarmdecoder. In other words, I have totally left the project.  Can somebody else please provide the second review?
Thanks!

**Comment by bobadair** [comment]
Hi @berndpfrommer. Do you want me to remove you from the CODEOWNERS file for this binding? If you're in there it will automatically add you as a reviewer for every PR, which you may not want if you don't have an AD anymore!

**Comment by berndpfrommer** [comment]
Yes, please remove me. Thanks! Thank you also for continuing to contribute
to openhab!

File: bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java
Comment (ddempsey3) [reply bundles/org.openhab.binding.alarmdecoder/src/main/java/org/openhab/binding/alarmdecoder/internal/protocol/KeypadMessage.java]: @bobadair Done!

**Comment by bobadair** [comment]
Thanks @ddempsey3! LGTM

**Review by Hilbrand — APPROVED** [review]
LGTM

**Comment by Hilbrand** [comment]
@ddempsey3  you need to sign off with your real name and e-mail address (see https://next.openhab.org/docs/developer/contributing.html#sign-your-work). If you add it to this pr (or edit the pr text) I'll then merge this pr. Thank you.

**Comment by cpmeister** [comment]
I'm sorry but we don't allow `users.noreply.github.com` email addresses, so you just need to give me a proper email address I can use for the signoff then I can merge.

Just add a comment with the email address you want to use.

Thanks for the contribution!

**Review by Hilbrand — CHANGES_REQUESTED** [review]
@ddempsey3 can you take care of the sign-off so we can have this merged for 2.5.9

**Comment by bobadair** [comment]
@ddempsey3 If you are concerned about spam, just create a new email address somewhere for github use.

**Comment by kaikreuzer** [comment]
I'm afraid we won't get this into 2.5.9 unless we hear from @ddempsey3 in the next few hours. I'll therefore remove the 2.5.9 milestone assignment for now.

**Review by kaikreuzer — APPROVED** [review]
Looking at the PR, I'd think we can treat this as a "small patch exception" and not require a correct sign-off for it.
@ddempsey3 for future PRs it would be great if you could make sure to have a correct sign-off line - thank you!

**Pull request merged**
